Components/Avatar/avatar.css

```diff
@@ -0,0 +1,40 @@
+.avatar {
+  border-radius: 50%;
+  padding: 2px;
+  border: 1px solid var(--violet-color);
+  background-color: transparent;
+}
+
+.avatar-img {
+  width: 100%;
+  height: 100%;
+  border-radius: inherit;
+}
+
+.avatar-txt {
+  background-color: var(--yellow-color-light);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
```

Comment:
A common CSS class can be made for display flex, justify center, align-items center instead of repeating it. 

Overall LGTM.